sky/spot/recovery_strategy.py

```diff
@@ -118,6 +118,9 @@ def terminate_cluster(self, max_retry: int = 3) -> None:
                                        f'{self.cluster_name}.') from e
                 logger.error('Failed to terminate the spot cluster '
                              f'{self.cluster_name}. Retrying.')
+                import traceback  # pylint: disable=import-outside-toplevel
+                logger.error(f'  Exception: {type(e)}: {str(e)}')
```

Comment:
`common_utils.format_exception` should be useful.

Reply:
Thanks! good point